exts/omni.example.spawnPrims/tutorial/tutorial.md

```diff
@@ -1,171 +1,144 @@
+    |------ |------ |
 
-| ![spawnprim_tut_png2](images/spawnprim_tutorial2.png) | ![spawnprim_tut_png3](images/spawnprim_tutorial3.png) |
-|------ |------ |
 
+From here another window will popup prompting on where to place the folder. For this example you will create it in the default location. 
 
-From here another window will popup prompting on where to place the folder. For this example we will create it in the default location. 
+- Once the save location is selected, hit `Create`
 
-```Step 1.2.2:``` Once the save location is selected, hit `Create`
+    ![spawnprim_tut_png4](images/spawnprim_tutorial4.png)
 
-![spawnprim_tut_png4](images/spawnprim_tutorial4.png)
+It will prompt for a Project Folder Name and Extension Name. 
 
-Then it will prompt for a Project Folder Name and Extension Name. 
+- Enter a Project Folder Name and Extension Name.
 
-```Step 1.2.3:``` Enter a Project Folder Name and Extension Name.
-
-For this example we chose `kit-exts-spawnPrims` for the Project Folder Name and `omni.spawn.primitive` for our Extension Name.
-
-> 📝 **Note:** Choose the name of an extension differently for publishing. We suggest ```companyName.extdescrip1.descrip2``` 
+- For this example choose `kit-exts-spawnPrims` for the Project Folder Name and `omni.spawn.primitive` for the Extension Name.
 
 | ![spawnprim_tut_png5](images/spawnprim_tutorial5.png) | ![spawnprim_tut_png6](images/spawnprim_tutorial6.png) |
 | --- | --- |
 
+> 📝 **Note:** Choose the name of an extension differently for publishing. For Example: ```companyName.extdescrip1.descrip2``` 
 
-<br><br>
-
-> 📝 **Note:** To learn more about the Extension Manager head over to our documentations [here](https://docs.omniverse.nvidia.com/app_code/prod_extensions/ext_extension-manager.html?). 
-
-<br><br>
-
+After choosing the Extension Name two things will happen.
 
+- Visual Studio Code will start up displaying the template code
+- A new window will appear called "My Window" in Omniverse
 
-After choosing the Extension Name two things will happen.
-1. Visual Studio Code will start up displaying the template code
-2. A new window will appear called "My Window" in Omniverse
 
-| ![spawnprim_tut_png7](images/spawnprim_tutorial7.png) | ![spawnprim_tut_png8](images/spawnprim_tutorial8.png) |
-| ------ | ------ |
+    | ![spawnprim_tut_png7](images/spawnprim_tutorial7.png) | ![spawnprim_tut_png8](images/spawnprim_tutorial8.png) |
+    | ------ | ------ |
 
-Now that we created a new extension template, there will be a ```Click Me``` button. When clicked, a message in the Console Window in Omniverse Code recording that the button was clicked. We will use this button later to spawn a primitive.
+Now that you created a new extension template, there will be a ```Click Me``` button. When clicked, a message in the Console Window in Omniverse Code recording that the button was clicked. You will use this button later to spawn a primitive.
```

Comment:
When clicked, an output message will appear in the Console Window in Omniverse Code stating that the button was clicked.